src/emu/services/src/audio/keysound/keysound.cpp

```diff
@@ -345,7 +345,7 @@ namespace eka2l1 {
     }
 
     void keysound_session::lock_context(eka2l1::service::ipc_context *ctx) {
-        ctx->complete(epoc::error_none);
```

Comment:
This just break its meaning. Please just let code be stubbed as it is

Reply:
@pent0 check it https://github.com/EKA2L1/EKA2L1/commit/e69502495791ac4f7c99fa21d58ad16deb6e435e